### PR TITLE
Cleans up the structure for the web app related docs

### PR DESCRIPTION
**Pull request by memasdeligeorgakis**
This solves https://github.com/anoma/specs/issues/102

### CODE REVIEW

**Comment by memasdeligeorgakis** [comment]
I see strange rendering at the end of a few pages, such as here: `/architecture/clients/web-wallet/client-application.html`. @tzemanovic or @jurevans do you know what could cause this?

image

File: src/SUMMARY.md [hunk 1]
```diff
@@ -8,8 +8,14 @@
...
-    - [Web wallet interface](./architecture/clients/web-wallet-interface.md)
-    - [Web explorer interface](./architecture/clients/web-explorer-interface.md)
+    - [Web Wallet UI and Features](./architecture/clients/user-interfaces.md)
+    - [Explorer UI and Features](./architecture/clients/web-explorer-interface.md)
+    - [IBC](./architecture/clients/web-wallet/ibc.md) 
+    - [Client Application](./architecture/clients/web-wallet/client-application.md) 
+    - [Key Derivation](./architecture/clients/web-wallet/key-derivation.md) 
+    - [Persistence](./architecture/clients/web-wallet/persistence.md) 
+    - [RPC](./architecture/clients/web-wallet/rpc.md) 
+    - [Transparent Transactions](./architecture/clients/web-wallet/transparent-transactions.md)
```
Comment (tzemanovic) [inline src/SUMMARY.md:18]: let's organize it like this to follow the dir structure
```suggestion
    - [Web Wallet UI](./architecture/clients/web-wallet.md)
      - [IBC](./architecture/clients/web-wallet/ibc.md) 
      - [Client Application](./architecture/clients/web-wallet/client-application.md) 
      - [Key Derivation](./architecture/clients/web-wallet/key-derivation.md) 
      - [Persistence](./architecture/clients/web-wallet/persistence.md) 
      - [RPC](./architecture/clients/web-wallet/rpc.md) 
      - [Transparent Transactions](./architecture/clients/web-wallet/transparent-transactions.md)
    - [Explorer UI and Features](./architecture/clients/web-explorer-interface.md)
```

**Comment by tzemanovic** [comment]
> I see strange rendering at the end of a few pages, such as here: `/architecture/clients/web-wallet/client-application.html`. @tzemanovic or @jurevans do you know what could cause this?
> 
> image

yeah, it's katex plugin, see #91

**Comment by jurevans** [comment]
> 

@tzemanovic I see the same thing in other docs (we haven't intended to use any Katex formatting). Thanks for catching that!

**Review by tzemanovic — CHANGES_REQUESTED** [review]
there are some dead links found in the CI:

``` 
error: File not found: ../architecture/clients/web-wallet-interface.md
   ┌─ releases/v1.md:80:3
   │
80 │ - [Web-based wallet](../architecture/clients/web-wallet-interface.md)
   │   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File not found: ../architecture/clients/web-wallet-interface.md
error: File not found: ../architecture/clients/web-explorer-interface.md
   ┌─ releases/v1.md:81:3
   │
81 │ - [Web-based block explorer](../architecture/clients/web-explorer-interface.md)
   │   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File not found: ../architecture/clients/web-explorer-interface.md
```

**Review by tzemanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**